src/components/Error.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const Error = ({ errorMessage }) => {
+  return (
+    <div>
+      <p style={{ color: 'red' }}>
+        <span role="img" aria-label="red X">
+          ❌
+        </span>{' '}
+        <b>{errorMessage}</b>
+      </p>
+    </div>
+  );
+};
+
```

Comment:
Here we used @rachelrohrbach and @veeteeran 's `Duplicate` component with a little refactoring

Reply:
This is great 💯 We're implementing this in our re-worked code

Reply:
Nice! This not only cuts down on code, but would also make the error messages throughout the app have the same look and feel. Ginormous thumbs up.

Reply:
Yep...love this.

Reply:
Rachel and Viet, you set it up so perfectly! That made it easy to use for our component, so thank you 👍